css/main.css

```diff
@@ -307,21 +307,134 @@ li {
 }
 
 .list-title {
-  font-family: "Inter", sans-serif;
+  font-family: 'Inter', sans-serif;
   font-style: normal;
   font-weight: bold;
   font-size: 32px;
   line-height: 44px;
 }
 
 .languages {
-  list-style: inside url("../media/svg/rectangle.svg");
+  list-style: inside url('../media/svg/rectangle.svg');
 }
 
 .frameworks {
-  list-style: inside url("../media/svg/square.svg");
+  list-style: inside url('../media/svg/square.svg');
 }
 
 .skills {
-  list-style: inside url("../media/svg/circle.svg");
+  list-style: inside url('../media/svg/circle.svg');
+}
+
+.contact-section {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  flex-direction: column;
+  margin-bottom: 50px;
+}
```

Comment:
- [ ] Please try as much as possible to stick to the design you chose on the [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1) as it is a project requirement.
### your buttons
![image](https://user-images.githubusercontent.com/98453979/184140056-4dced22b-d0bf-4c92-869a-c339dcd524de.png)

### figma buttons
![image](https://user-images.githubusercontent.com/98453979/184140538-41a74bcf-b75d-4c0b-9147-fc91116732e7.png)

### consider the spacing of elements
![image](https://user-images.githubusercontent.com/98453979/184141194-065f9bd0-c65f-48d4-8d4c-9af4a14dec5d.png)

### figma design

![image](https://user-images.githubusercontent.com/98453979/184141738-c165de9a-c0b2-4c90-89ba-62c0c22fd7cf.png)

### your footer
![image](https://user-images.githubusercontent.com/98453979/184142464-34343735-0950-421c-bffe-980faf995d08.png)

### figma footer
![image](https://user-images.githubusercontent.com/98453979/184142683-ee61fbb2-b17f-4ff4-813f-cec34fbe999e.png)